android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -295,14 +295,13 @@ protected void onResume() {
       switch (data.getScheme().toLowerCase()) {
         // Android DownloadManager
         case "content":
-          checkStoragePermission(data);
-          break;
         // Chrome downloads and Filebrowsers
```

Comment:
The removal of the two lines above makes sense but the comment masks the fallthrough effect; can you tidy that up?